portfolio/src/main/java/com/google/sps/data/MergeBooks.java

```diff
@@ -20,7 +20,7 @@ public static Book merge(Book googleApiBook, Book goodReadsBook) {
     validate(goodReadsBook);
     validateISBN(googleApiBook.isbn());
     Preconditions.checkArgument(
-        googleApiBook.title().equals(goodReadsBook.title()), "The books must be of the same title");
+        googleApiBook.title().toLowerCase().equals(goodReadsBook.title().toLowerCase()), "The books must be of the same title. Expected [" + goodReadsBook.title() + "] but got ["+googleApiBook.title()+"]");
```

Comment:
nit: this line is super long. Not sure how to change it, but I'm sure a formatter would break it into a few lines.